test/fit_tests/deploy/rackhd_package_install.py

```diff
@@ -168,57 +228,30 @@ def test03_install_rackhd_config_files(self):
-            "wget 'https://dl.bintray.com/rackhd/binary/ipxe/monorail.ipxe';"
-        )['exitcode'], 0, "Binary Support Install failure.")
-        self.assertEqual(fit_common.remote_shell(
-            ENVVARS +
-            "mkdir -p /var/renasar/on-http/static/http/common;"
-            "cd /var/renasar/on-http/static/http/common;"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/base.trusty.3.16.0-25-generic.squashfs.img';"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/discovery.overlay.cpio.gz';"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/initrd.img-3.16.0-25-generic';"
-            "wget 'https://dl.bintray.com/rackhd/binary/builds/vmlinuz-3.16.0-25-generic';"
-        )['exitcode'], 0, "Binary Support Install failure.")
-
-    def test05_reboot_and_check(self):
-        print "**** Reboot and check installation."
-        self.assertEqual(fit_common.remote_shell(
-            "touch /etc/default/on-dhcp-proxy /etc/default/on-http /etc/default/on-tftp /etc/default/on-syslog /etc/default/on-taskgraph"
-        )['exitcode'], 0, "Install failure.")
-        # reboot
-        print "**** Rebooting appliance..."
-        fit_common.remote_shell("shutdown -r now")
-        print "**** Waiting for login..."
-        fit_common.countdown(30)
-        shell_data = 0
-        for dummy in range(0, 30):
-            shell_data = fit_common.remote_shell("pwd")
-            if shell_data['exitcode'] == 0:
-                break
+    def test06_startup(self):
+        print "Restart services."
+        self.assertEqual(fit_common.remote_shell("service isc-dhcp-server restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-http restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-dhcp-proxy restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-syslog restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-taskgraph restart")['exitcode'], 0, "on-http failure.")
+        self.assertEqual(fit_common.remote_shell("service on-tftp restart")['exitcode'], 0, "on-http failure.")
+        print "**** Check installation."
+        for dummy in range(0, 10):
+            try:
+                fit_common.rackhdapi("/api/2.0/config")
+            except:
+                fit_common.time.sleep(10)
             else:
-                fit_common.time.sleep(5)
-        self.assertEqual(shell_data['exitcode'], 0, "Shell test failed after appliance reboot")
-        fit_common.time.sleep(10)
+                break
         self.assertEqual(fit_common.rackhdapi("/api/2.0/config")['status'], 200, "Unable to contact RackHD.")
 
 if __name__ == '__main__':
-    fit_common.unittest.main()
\ No newline at end of file
+    fit_common.unittest.main()
```

Comment:
Not a good idea to import unittest from fit_common unless you are overriding it.  This should be imported within this module.